squad/frontend/templates/squad/build-nav.html

```diff
@@ -18,11 +18,17 @@ <h1>
             All test results
         </a>
     </li>
+    {% if build.test_jobs.all %}
```

Comment:
you might want to use `count` instead of `all` to avoid fetching the test jobs since you only want to know if there are any

Reply:
good point